react/utils/normalize.ts

```diff
@@ -84,14 +229,35 @@ const defaultSeller = { commertialOffer: { Price: 0, ListPrice: 0 } }
 const resizeImage = (url: string, imageSize: string | number) =>
   changeImageUrlSize(toHttps(url), imageSize)
 
+/**
+ * @description
+ * This method is responsible of constructing the product type for the product-context.
+ * The property `product.sku` is built by these specifications
```

Comment:
Grammarly attacks again haha! Feel free to argue against 
```suggestion
 * @description
 * This method is responsible for constructing the product type for the product-context.
 * The property `product.sku` is built by these specifications
```